openstack_controller/tests/test_config.py

```diff
@@ -0,0 +1,123 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import logging
+import re
+
+import pytest
+
+from datadog_checks.openstack_controller import OpenStackControllerCheck
+
+from .common import CHECK_NAME, TEST_OPENSTACK_NO_AUTH_CONFIG_PATH
+
+
+@pytest.mark.parametrize(
+    'instance, exception_msg',
+    [
+        pytest.param({'user': {'domain': {'id': 'test_id'}}}, 'Missing name', id='empty'),
+        pytest.param({'user': {'domain': {'id': 'test_id'}}}, 'Missing name', id='empty name'),
```

Comment:
What is the difference between this and the previous one?

Reply:
Good catch!